docs/content/latest/yugabyte-cloud/cloud-security/_index.md

```diff
@@ -1,10 +1,10 @@
 ---
-title: Yugabyte Cloud architecture
-headerTitle: Yugabyte Cloud architecture
-linkTitle: Yugabyte Cloud architecture
-description: Security architecture in Yugabyte Cloud.
+title: Security architecture
+headerTitle: Security architecture
+linkTitle: Security architecture
+description: Security architecture of Yugabyte Cloud.
 image: /images/section_icons/index/secure.png
-headcontent: Yugabyte Cloud architecture and security.
+headcontent: Yugabyte Cloud infrastructure security architecture.
```

Comment:
"... infrastructure security architecture" doesn't sound good. Security architecture generally includes a security aspect of infrastructure as well,